crates/readme.md

```diff
@@ -0,0 +1,44 @@
+# Structure
+
+Below we briefly describe the purpose of each crate in this repository.
+
+## acir - Abstract Circuit Intermediate Representation
+
+This is the intermediate representation that Noir compiles down to. It is agnostic to any particular NP-Complete language.
```

Comment:
Is this really true? I didn't think our circuits were NP complete at all. How can we encode an infinite loop for example? I also doubt that we can really claim it is agnostic to "any" language whether NP-complete or not since we may lack necessary primitives for any (e.g. IO).

Reply:
Yep, we can trivially reduce the arithmetic expressions part of ACIR to a known np-complete (NPC) language known as R1CS.

I'm not sure what IO has to do with ACIR being agnostic to NPC languages; the distinction, I'm drawing here is that ACIR is not geared towards any particular NPC representation that we are familiar with like R1CS or CSAT (Plonkish)